src/Apex-Assignment1/classes/Collection.cls

```diff
@@ -0,0 +1,68 @@
+/*
+Purpose        : This class has been created to practice Maps,List and Set.
+Created Date   : 11/25/2024
+Created By     : Sarita Rathore
+Revision_Log   : V1.0
+*/
+public with sharing class Collection {
+/*
+Description    : This method gives the employees in each department.
```

Comment:
@Sarita-Rathore-1306 only add one line description for the method . We don't need to add date and other this whenw e add method description